doc/source/whatsnew/v0.13.0.rst

```diff
@@ -153,14 +153,18 @@ API changes
 
   Added the ``.bool()`` method to ``NDFrame`` objects to facilitate evaluating of single-element boolean Series:
 
-  .. ipython:: python
+  .. code-block:: python
 
-     pd.Series([True]).bool()
-     pd.Series([False]).bool()
-     pd.DataFrame([[True]]).bool()
-     pd.DataFrame([[False]]).bool()
+     >>> pd.Series([True]).bool()
+        ..  True
```

Comment:
No indentation on the output, otherwise looks good.

```
>>> pd.Series([True]).bool()
True
```

Reply:
Done